huggingfaceserver/rockcraft.yaml

```
@@ -0,0 +1,77 @@
+# Based on https://github.com/kserve/kserve/blob/v0.13.0/python/huggingface_server.Dockerfile
+name: huggingfaceserver
+summary: Huggingface server for Kserve deployments
+description: "Kserve Huggingface server"
+version: "0.13.0"
+license: Apache-2.0
+base: ubuntu@22.04
+run-user: _daemon_
```

We probably want to add the environment variables defined in [these lines](https://github.com/kserve/kserve/blob/1c51eeee174330b076e4171e6d71e9138f2510b3/python/huggingface_server.Dockerfile#L55-L60).
```suggestion
run-user: _daemon_
environment: 
  HF_HOME: "/tmp/huggingface"
  SAFETENSORS_FAST_GPU: "1"
  HF_HUB_DISABLE_TELEMETRY: "1"
```